helm-charts/falcon-sensor/README.md

```diff
@@ -127,7 +127,8 @@ The following tables lists the more common configurable parameters of the chart
 | `node.image.pullSecrets`          | Pull secrets for private registry                                      | None       (Conflicts with node.image.registryConfigJSON)               |
 | `node.image.registryConfigJSON`   | base64 encoded docker config json for the pull secret                  | None       (Conflicts with node.image.pullSecrets)                      |
 | `node.daemonset.resources`        | Configure Node sensor resource requests and limits (eBPF mode only)    | None       (Minimum setting of 250m CPU and 500Mi memory allowed). Default for GKE Autopilot is 750m CPU and 1.5Gi memory.<br><br><div class="warning">:warning: **Warning**:<br>If you configure resources, you must configure the CPU and Memory Resource requests and limits correctly for your node instances for the node sensor to run properly!</div> |
-| `falcon.cid`                      | CrowdStrike Customer ID (CID)                                          | None       (Required)                                                   |
+| `node.cdpRolesEnabled`            | Enabled the Cloud Data Protection module                               | `true`                                                                  |
```

Comment:
```suggestion
| `node.cdpRolesEnabled`            | Enable cluster roles for Cloud Data Protection module                               | `true`                                                                  |
```
Since this flag only allows the role to be created. Additional config changes(Falcon UI) are required to enable the protection module.